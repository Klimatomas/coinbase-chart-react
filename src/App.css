.popup {
  min-height: 50px;
}

.row {
  display: flex;
  justify-content: center;
 /* align-items: center; /*here?*/
}
#data-container {
  display: flex;
  flex-direction: row;
  justify-content: space-between;
  align-content: center;
}

.box > div:last-child {
  background-color: rgba(235, 235, 235, .8);
  border: 1px solid rgb(220, 220, 220);
  border-radius: 3px;
  padding: 0 10px;
}