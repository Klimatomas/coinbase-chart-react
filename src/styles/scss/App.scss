@import '_variables.scss';

.popup {
  min-height: 5rem;
}

.row {
  display: flex;
  justify-content: center;
}

#data-container {
  background-image: linear-gradient($grimmy_grey , $dim_grey, $dark_grey, $dark_grey, $coal);
  display: flex;
  flex-direction: row;
  justify-content: space-evenly;
  align-content: center;
  border: .2rem solid $dim_grey;
  border-radius: .5rem;
}

.box {
  & > div {
    &:first-child {
      -webkit-text-stroke: .5px $black;
    }

    &:last-child {
      width: 100%;
      background-color: $light_grey;
      border: 1px solid $silver_grey;
      border-radius: .3rem;
      padding: .5rem 1rem;
      color: $ghostwhite;
      box-shadow: inset 0 .2rem 0 $midnight_blue, 0 .2rem .5rem $black;
      text-align: center;
      transition: all 1.5s ease-in;

      &:hover {
        background-color: rgb(0, 175, 175);
        box-shadow: inset 0 .2rem 0 rgb(0, 96, 96), 0 .2rem .5rem $black;
      }
    }
 }
}
