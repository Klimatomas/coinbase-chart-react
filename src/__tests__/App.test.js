import React from 'react';
import { shallow } from 'enzyme';
import App from '../App';

test('renders <App /> correctly', () => {
  const wrapper = shallow(<App />);
  expect(wrapper.find(App)).toBeDefined;
  expect(wrapper).toMatchSnapshot();
});
