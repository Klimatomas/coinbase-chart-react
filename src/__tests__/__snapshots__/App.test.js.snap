// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <App /> correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <App />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="row"
        >
          <div
            className="infobox"
          />
        </div>,
        <div
          className="row"
        >
          <div
            className="popup"
          />
        </div>,
        <div
          className="row"
        >
          <div
            className="chart"
          />
        </div>,
      ],
      "className": "container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="infobox"
          />,
          "className": "row",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": null,
            "className": "infobox",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="popup"
          />,
          "className": "row",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": null,
            "className": "popup",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="chart"
          />,
          "className": "row",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": null,
            "className": "chart",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="row"
          >
            <div
              className="infobox"
            />
          </div>,
          <div
            className="row"
          >
            <div
              className="popup"
            />
          </div>,
          <div
            className="row"
          >
            <div
              className="chart"
            />
          </div>,
        ],
        "className": "container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <div
              className="infobox"
            />,
            "className": "row",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": null,
              "className": "infobox",
            },
            "ref": null,
            "rendered": null,
            "type": "div",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <div
              className="popup"
            />,
            "className": "row",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": null,
              "className": "popup",
            },
            "ref": null,
            "rendered": null,
            "type": "div",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <div
              className="chart"
            />,
            "className": "row",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": null,
              "className": "chart",
            },
            "ref": null,
            "rendered": null,
            "type": "div",
          },
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
